.github/workflows/ci.yml

```
@@ -233,6 +233,7 @@ jobs:
           LAST_FAILED_RUN: ${{ vars.LAST_FAILED_RUN }}
           GITHUB_BASE_SHA: ${{ github.event.pull_request.base.sha }}
           WC_VERSION: ${{ inputs.wc-version }}
+          PLUGIN_SLUG: ${{ inputs.refName == 'nightly' && 'woocommerce-trunk-nightly' || '' }} # required by test:plugincheck
```

Optional, but would it be worth it to explicitly set the fallback to the default `woocommerce` so we don't need to provide defaults anywhere else if this gets reused?

```suggestion
          PLUGIN_SLUG: ${{ inputs.refName == 'nightly' && 'woocommerce-trunk-nightly' || 'woocommerce' }} # required by test:plugincheck
```